nautobot/dcim/views.py

```diff
@@ -1189,6 +1290,202 @@ class DeviceBayTemplateBulkDeleteView(generic.BulkDeleteView):
     filterset = filters.DeviceBayTemplateFilterSet
 
 
+#
+# Module bay templates
+#
+
+
+class ModuleBayCommonViewSetMixin:
+    def create(self, request, *args, **kwargs):
```

Comment:
Mind adding a docstring or comment explaining how this is different than the usual viewset `create` implementation?